src/classes/FORM_PermissionValidator.cls

```diff
@@ -0,0 +1,146 @@
+public with sharing class FORM_PermissionValidator {
+
+    public enum AccessLevel {
+        VALIDATE_READ, VALIDATE_CREATE, VALIDATE_UPDATE
+    }
+    
+    private final String OBJECT_CRUD_ERROR = 'CRUD';
```

Comment:
Can we make these two static an public so we can use them in the test class as well.

Reply:
fixed